### DISCUSSION TITLE
Cannot execute create-next-app because of permission

### DISCUSSION THREAD

**Discussion by matamatanot** [vercel/next.js#14744] [Ideas]
# Bug report

## Describe the bug

Cannot execute create-next-app in`/User`(Not Allowed) because of **permission**.

## To Reproduc

1. Go to `/Users`
2. `npx create-next-app --example hello-world hello-world-app` (any example)
3. See error

## Expected behavior

Explicitly indicate that the error is due to permission.

## Screenshots

<img width="598" alt="console" src="https://user-images.githubusercontent.com/39780486/86164228-5d925080-bb4c-11ea-9def-587f68d38caf.png">

## System information

- OS: macOS
- Version of Node.js: 12.7.0

## Additional context

An example of npm

```
npm ERR! path /Users/package.json
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall open
npm ERR! Error: EACCES: permission denied, open '/Users/package.json'
npm ERR!  [Error: EACCES: permission denied, open '/Users/package.json'] {
npm ERR!   stack: "Error: EACCES: permission denied, open '/Users/package.json'",
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'open',
npm ERR!   path: '/Users/package.json'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator (though this is not recommended).
```

**Comment by jamesmosier** [comment]
That is a protected folder. Instead try running `create-next-app` inside a folder such as `Documents` where your user profile has permissions.

**Reply by matamatanot** [reply]
Oh, I thought I had post Issue but I had posted Discussions.
It's exactly as you say. 

But, I think it's more user-friendly to explicitly warn people.

**Reply by jamesmosier** [reply]
That warning would have to come from npm itself, Next cannot control that because it happens before `create-next-app` is invoked.

**Comment by IRediTOTO** [comment]
This come from your OS not nextjs, let's try google to fix this.

**Comment by matamatanot** [comment]
#14889

**Comment by CodyBontecou** [comment]
I recently ran into this with the command `npx create-next-app@latest shadcn/ui --typescript --tailwind --eslint`

The issue is in the `shadcn/ui`. Changing this to `shadcn-ui` fixed my issue. So the appropriate command is:

`npx create-next-app@latest shadcn-ui --typescript --tailwind --eslint`
